pipelined/bevy_render2/src/render_graph/graph.rs

```diff
@@ -58,6 +58,16 @@ impl RenderGraph {
         id
     }
 
+    pub fn remove_node(&mut self, name: impl Into<Cow<'static, str>>) {
```

Comment:
`remove_node` will leave edges that referenced the node around if there were any. For `A -> B` `graph.remove("A")` you'd need to remove the `A` node, and remove all `input_edges` (I think) of `B` that reference `A`.

Reply:
Adding a slot edge or node edge from an output to an input creates an Edge of the appropriate type and adds it to both the output node's output edges and the input node's input edges. As such, I think what needs to be done is to look at all the input edges for the node, and remove that edge from the edge's output node's output edges, and look at all the output edges for the node, and remove that edge from the edge's input node's input edges. I made this change.